README.md

```diff
@@ -1,71 +1,152 @@
-# Fury Kubernetes OPA
+<h1>
+    <img src="https://github.com/sighupio/fury-distribution/blob/master/docs/assets/fury-epta-white.png?raw=true" align="left" width="90" style="margin-right: 15px"/>
+    Kubernetes Fury OPA
+</h1>
 
-Fury Kubernetes OPA provides a policy engine based on OPA Gatekeeper to enable custom policy enforcement in a
-Kubernetes Cluster.
+![Release](https://img.shields.io/github/v/release/sighupio/fury-kubernetes-opa?label=Latest%20Release)
+![License](https://img.shields.io/github/license/sighupio/fury-kubernetes-opa?label=License)
+![Slack](https://img.shields.io/badge/slack-@kubernetes/fury-yellow.svg?logo=slack&label=Slack)
 
-## OPA Packages
+<!-- <KFD-DOCS> -->
 
-The following packages are included in the Fury Kubernetes OPA module:
+**Kubernetes Fury OPA** provides a policy engine based on OPA Gatekeeper to enable enforcement of custom policies in a Kubernetes Cluster for the [Kubernetes Fury Distribution (KFD)][kfd-repo] via a [Validating Admission Webhook][kubernetes-vaw-docs].
```

Comment:
What do you think of simplifying it a little bit?

Something on the line of:

**Kubernetes Fury OPA** provides policy enforcement for the [Kubernetes Fury Distribution (KFD)][kfd-repo] using OPA Gatekeeper.

Reply:
much better!